### PR TITLE
P1673R3 PR2: linalg namespace, function renames, wording fixes

### PR DESCRIPTION
**Pull request by mhoemmen**
@crtrott

### CODE REVIEW

**Review by dalg24 — APPROVED** [review]

**Comment by mhoemmen** [comment]
@dalg24 Hi Damien -- thanks for the approval! : - D

**Comment by dalg24** [comment]
I do not have permissions

File: D1673/blas_interface.md [hunk 1]
```diff
@@ -698,6 +821,50 @@ a special case of tensor algebra, users of linear algebra have
...
+from the [Kokkos library](https://github.com/kokkos/kokkos),
+and `Kokkos::View` directly inspired `basic_mdspan`.
+The C++ Standard does not currently define how such memory behaves,
+but implementations could define its behavior and make it work with `basic_mdspan`.
+This would, in turn, let implementations define our algorithms
+to operate on such memory efficiently,
+if given the right implementation-specific `ExecutionPolicy`.
+
+Our proposal excludes algorithms that might write to their output arguments
+at some time after after the algorithm returns.
```
Comment (crtrott) [inline D1673/blas_interface.md:854]: twice after

File: D1673/blas_interface.md [hunk 2]
```diff
@@ -698,6 +821,50 @@ a special case of tensor algebra, users of linear algebra have
...
+might write its scalar result via a pointer to GPU memory,
+instead of returning the result "on the CPU."
+
+Nothing in principle prevents `basic_mdspan` from viewing memory
+that is inaccessible from ordinary C++ code.
+This is a major feature of the `Kokkos::View` class
+from the [Kokkos library](https://github.com/kokkos/kokkos),
+and `Kokkos::View` directly inspired `basic_mdspan`.
+The C++ Standard does not currently define how such memory behaves,
+but implementations could define its behavior and make it work with `basic_mdspan`.
```
Comment (crtrott) [inline D1673/blas_interface.md:848]: Maybe add something like: For example one could envision in the future accessors which are memory space specific (a pretty direct analogy to allocators) which define which type of execution agents can access them.

**Review by crtrott — APPROVED** [review]
Just left two comments we may wanna address later.

**Pull request merged**